lib/naver/searchad/api/adgroup/service.rb

```diff
@@ -5,35 +5,55 @@ module Adgroup
         class Service < Naver::Searchad::Api::Core::BaseService
 
           def initialize
-            super('https://api.naver.com/', 'ncc/adgroups/')
+            super('https://api.naver.com/', 'ncc/')
```

Comment:
even I put them in constants, they will be used only once. What about adding `freeze` to them? at least, they will be immutable :)